배수연/4월 2주차/BOJ_S2_5525.java

```diff
@@ -0,0 +1,64 @@
+package boj;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class BOJ_S2_5525 {
+
+	public static void main(String[] args) throws Exception {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		int N = Integer.parseInt(br.readLine());
+		int M = Integer.parseInt(br.readLine());
+		String str = br.readLine();
+		int ans = 0;
+		
+		int idx = 0, cnt = 0;
+		while(idx<M-2) {
+			String temp = str.substring(idx, idx+3);
+			if(temp.equals("IOI")) {
+				cnt++;
+				if(cnt == N) {
+					cnt--;
+					ans++;
+				}
+				idx++;
+			}
+			else cnt = 0;
+			idx++;
+		}
```

Comment:
이 부분 idx 를 인자로 하는 for 문으로 작성하셨으면 더 가독성이 좋았을 것 같아요 !
```suggestion
		for(idx = 0 ; idx < M - 2 ; idx++) { ... }
```